sql/set.go

```diff
@@ -94,7 +94,7 @@ func (p *planner) SetTimeZone(n *parser.SetTimeZone) (planNode, *roachpb.Error)
 			location = "UTC"
 		}
 		if _, err := time.LoadLocation(location); err != nil {
-			return nil, roachpb.NewError(err)
+			return nil, roachpb.NewUErrorf("cannot find %s: %v", location, err)
```

Comment:
Are you sure about %s for error? I've seen it both ways, so I double checked before pushing this and it seems like Docker uses %v for errors. So does http://blog.golang.org/error-handling-and-go.